src/app/app.component.ts

```diff
@@ -281,7 +281,7 @@ export class AppComponent {
     /* eslint-disable @typescript-eslint/prefer-for-of */
     const acc = document.getElementsByClassName('accordion');
     for (let i = 0; i < acc.length; i++) {
-      acc[i].addEventListener('click', function() {
+      acc[i].addEventListener('click', function () {
```

Comment:
At least one previous lint configuration would consider a space between the `function` keyword and the open parenthesis a linting error.